.release-it.json

```diff
@@ -0,0 +1,11 @@
+{
+  "git": {
+    "commitMessage": "release: v${version}",
+    "tagName": "v${version}",
+    "tagAnnotation": "Release v${version}"
+  },
+  "github": {
+    "release": true,
+    "releaseName": "v${version}"
```

Comment:
acho que aqui teria que configurar o tokenRef https://github.com/ContaAzul/ca-design-system/blob/master/release-it.config.js#L12